### DISCUSSION TITLE
Is this what is supposed to happen with text strokes? Or its a glitch?

### DISCUSSION THREAD

**Discussion by VeiloPot** [keiwando/toonsquid#330] [Q&A]
Whenever im deciding to put strokes on texts, it does this weird sharp shape thing under the letters and it mainly happens on the letters "M" and "V".

Is this whats supposed to happen?https://github.com/keiwando/toonsquid/assets/135170331/0c8a1324-6229-42d4-92b5-06f3de698f79

**Comment by Cat-Butler** [comment]
I think it is being caused by a default Miter join style on the acute angles when the outline width gets sufficiently high. I've seen a similar effect in a vector drawing app, for the same reason.

![join-sml](https://github.com/keiwando/toonsquid/assets/25494772/3a32cf76-1949-4d33-9b0c-bf19d420522f)

Unfortunately, while we can change join styles on hand-drawn Paths, ToonSquid does not [currently, at the time of writing] have an option to change the join style for text, nor to convert text to outlines. 

The only workarounds I can immediately think of are:
- use a different font without those acute angles
- use a thinner outline that does not extend so far on the angles
- create your text and outline in a different app (with more join style controls) and copy-paste or import it into ToonSquid

**Comment by Dewin02** [comment]
I also have the same problem when using text in the application, when increasing the thickness of the text stroke two things happen:
1. Some points appear in some texts.
2. The stroke width hides the text padding.

It would be nice to have the option to align the text stroke, that way you could place the stroke inside, center or outside.
It would also be nice to have the option to join styles in the texts.

https://github.com/keiwando/toonsquid/assets/112216968/de725e4a-2b94-4862-b70f-0c0d7f2631cc
